addons/cargo/functions/fnc_renameObject.sqf

```diff
@@ -0,0 +1,67 @@
+#include "script_component.hpp"
+#include "\a3\ui_f\hpp\defineResincl.inc"
+/*
+ * Author: JasperRab
+ * Shows rename UI and renames object once completed.
+ *
+ * Arguments:
+ * 0: Target <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [crate_7] call ace_cargo_fnc_renameObject
+ *
+ * Public: Yes
+ */
+
+params ["_target"];
+
+private _display = findDisplay IDD_MISSION createDisplay "RscDisplayEmpty";
+_display displayAddEventHandler ["Unload", {
+    params ["_display", "_exitCode"];
+    if (_exitCode isEqualTo 1) then { // only set if pressed on the Save button
+        private _ctrlEdit = (((allControls _display) select {ctrlClassName _x isEqualTo "RscEdit"}) select 0);
```

Comment:
A better way would be to just `setVariable` `_ctrlEdit` on the display and access it here using `getVariable`.